web/src/constants/SearchParams.ts

```
@@ -1,7 +1,27 @@
+
+export const Flow: string = "flow";
+
+export const FlowID: string = "flow_id";
+
+export const SubFlow: string = "subflow";
+
+export const UserCode: string = "user_code";
+
+export const Decision: string = "decision";
+
+export const Error: string = "error";
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)